packages/seedless-onboarding-controller/CHANGELOG.md

```diff
@@ -16,5 +16,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Add array of new seed phrase backups to the metadata store in batch (useful in multi-srp flow)
   - Fetch seed phrase metadata from the metadata store
   - Update the password of the seedless onboarding flow
+  - checkIsPasswordOutdated to check current password is outdated compare to global password
```

Comment:
You need to add a link to the pr just as I showed in the referenced pr. Also, I think you can’t just sunmarize the changes in one bullet point.